### PR TITLE
Add missing dependencies to README

### PR DESCRIPTION
**Pull request by maximbaz**
Let's cleanup dependencies list 🙂 

- I removed an empty line, not sure if that separation meant something?
- `chrono-date` and `spdlog` are required, I think in AUR package you should add them instead of letting meson download them
- `sway` is not really a dependency (especially since https://github.com/Alexays/Waybar/commit/6522a7acb4a87ecfa0f8e85a190b9743ba52a6f9) in a sense that sway package provides no shared libraries needed for the modules to work, it feels like adding a dependency on "linux"
- `wlroots` also seems to be not really a dependency, `ldd /usr/bin/waybar | grep wlroots` finds nothing
- `libappindicator-gtk3` should be a required dependency based on [this log message](https://github.com/Alexays/Waybar/blob/9817955fef845444a95092653ea202f75ba7c3e4/src/modules/sni/tray.cpp#L13), I think you should add it to meson.build as well to be honest
- I saw [PR adding pugixml](https://github.com/Alexays/Waybar/pull/661) as dependency, but I dont see it anywhere in the codebase being needed? UPDATE: I see now, will be needed after #659
- `wayland-protocols` is needed during the build as make dependency, for the xml definitions of protocols

----

For your AUR package:

- Move libmpdclient to `depends` (if you compile with mpd support, the shared libraries become required)
- Here's the current list of dependencies as I see it:

```
depends=(
    'gtkmm3'
    'libjsoncpp.so'
    'libinput'
    'libsigc++'
    'fmt'
    'wayland'
    'chrono-date'
    'libspdlog.so'
    'gtk-layer-shell'
    'libpulse'
    'libnl'
    'libappindicator-gtk3'
    'libdbusmenu-gtk3'
    'libmpdclient'
)
makedepends=(
    'git'
    'cmake'
    'meson'
    'scdoc' # For generating manpages
    'wayland-protocols'
)
optdepends=(
    'otf-font-awesome: Icons in the default configuration'
)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,21 +45,30 @@ $ waybar
...
 wayland
-wlroots
+chrono-date
+spdlog
 libgtk-3-dev [gtk-layer-shell]
 gobject-introspection [gtk-layer-shell]
 libgirepository1.0-dev [gtk-layer-shell]
 libpulse [Pulseaudio module]
 libnl [Network module]
-sway [Sway modules]
```
Comment (Alexays) [inline README.md:58]: Why removing sway?

Comment (maximbaz) [reply README.md:58]: I see this list being used by people who want to package waybar for different operating system, and having sway as a dependency says as if waybar cannot physically run without sway, as if it depends on some shared libraries from sway - which it doesn't. Same for wlroots dependency, as far as I could see waybar doesnt actually depend on anything from wlroots. More details in the description of the PR :)

**Comment by Alexays** [comment]
Thanks for the update!
I'll update the AUR package 😄

**Comment by Alexays** [comment]
And pugixml will be added once the PR will be merged :)

**Pull request merged**